ci/build/build-docker-images.py

```diff
@@ -111,11 +111,11 @@ def _get_commit_sha():
 def _configure_human_version():
     global _get_branch
     global _get_commit_sha
-    fake_branch_name = input(
+    fake_branch_name = str(input(
```

Comment:
If I understand correctly, input() doesn't have security issue in Python 3? Ray doesn't support Python 2.X so it should be fine.

Reply:
@jjyao input should be validation by type or by function input is safe

Reply:
Hmm, I'm not sure if I understand what you said. Do we still need to add `str(input())` if we are using Python 3?

Reply:
@jjyao this up to you , you review this https://cisserv1.towson.edu/~cssecinj/modules/cs0/cs0-input-validation-python/